src/Modules/OrchardCore.Commerce.Payment.Stripe/Views/CheckoutStripe.cshtml

```diff
@@ -45,7 +46,19 @@
     <script at="Head" asp-name="@ResourceNames.StripePaymentForm"></script>
     <script at="Foot" depends-on="StripeJs,StripePaymentForm">
         stripePaymentForm(
-            Stripe(@publishableKey.JsonHtmlContent()),
+            @if (!string.IsNullOrWhiteSpace(accountId))
+            {
+                <text>
+                Stripe(@publishableKey.JsonHtmlContent(), {
+                    stripeAccount: '@accountId',
+                }),
```

Comment:
Is it guaranteed that the `accountId` doesn't have an apostrophe in it? I think this would be safer:
```suggestion
                Stripe(@publishableKey.JsonHtmlContent(), @Json.Serialize(new { StripeAccount = accountId })),
```